hybrid/compartment_examples/inter_comp_call/secure_switch_compartment.S

```diff
@@ -7,45 +7,44 @@
 
 /*
  * The compartment switch function. Expects compartment information to be
- * stored in memory (defined by the capability stored in register `c1`).
+ * stored in memory (defined by the capability stored in register `c29`).
```

Comment:
I see all these changes away from `c29`, which as discussed, matters for the purposes of `ldpblr`. If there is no  top-level comment mentioning how this is a special register (again, for the purposes for `ldpblr`), perhaps one is in order.

Reply:
Yeah, I didn't mean to push [52e92e5](https://github.com/capablevms/cheri-examples/pull/75/commits/52e92e513db4d282cc399103626e0305787d88cc) (I reverted it in [8f67504](https://github.com/capablevms/cheri-examples/pull/75/commits/8f67504a0231e747a113fd3769da81add68329fe)). I can squash all of this away to make things easier to review.

Reply:
Not applicable as this change was reverted.